### PR TITLE
Update ref_language-server-protocol.adoc

### PR DESCRIPTION
**Pull request by DStatWriter**
Addition of link to Che4z docs

Signed-off-by: Daniel Statham <Daniel.Statham@Broadcom.com>

### What does this PR do?
Addition of link to Che4z doc supporting Language Support for COBOL

### What issues does this PR fix or reference?
#15303

### CODE REVIEW

File: src/main/pages/che-7/contributor-guide/ref_language-server-protocol.adoc [hunk 1]
```diff
@@ -13,4 +13,7 @@ summary:
 
 The Language Server Protocol (LSP) defines the protocol used between an editor or IDE and a language server that provides language features, including auto complete, go to definition and find all references. 
 
-For more details, see link:https://microsoft.github.io/language-server-protocol/[].
+For more details, see:
+
+- link:https://microsoft.github.io/language-server-protocol/[Microsoft Langauge Server Protocol].
+- link:https://www.eclipse.org/che/docs/che-7/eclipse-che4z/[Eclipse Che4z]
```
Comment (rkratky) [inline src/main/pages/che-7/contributor-guide/ref_language-server-protocol.adoc:19]: @DStatWriter, how is that related?

Comment (DStatPersonal) [reply src/main/pages/che-7/contributor-guide/ref_language-server-protocol.adoc:19]: Hi Robert, the Che4z package includes, among other things the Cobol Language support, and some additional functionality to be added, however if you don't feel there is a strong enough link I can remove my added link?

Comment (rkratky) [reply src/main/pages/che-7/contributor-guide/ref_language-server-protocol.adoc:19]: It would perhaps make sense to link from the Che4z docs to the LSP content, not vice versa.

**Review by themr0c — CHANGES_REQUESTED** [review]
I don't agree with this proposal. This reference file is about defining what is the Language Server Protocol itself. It is not about having a list of various implementations using this protocol.
